aws/v18.0.0-alpha1/README.md

```diff
@@ -0,0 +1,774 @@
+# :zap: Giant Swarm Release v18.0.0-alpha1 for AWS :zap:
+
+This release provides support for Kubernetes 1.23 and replaces the use of AWS CNI with Cilium. Release also upgrades most of running components which can be found further in release notes.
+
+> **_Warning:_** This is an **`alpha preview release`** intended only for testing Kubernetes v1.23 changes and replacement of AWS CNI with Cilium. Upgrading to or from this version is not supported.
```

Comment:
We need to point out a few notes about Cilium I think.
The main change to me is how NetworkPolicies work. More specifically network policies that reference pod or service cidrs won't work any more see https://docs.cilium.io/en/stable/concepts/kubernetes/policy/#networkpolicy-state